src/components/about/About.js

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
+  return (
+    <>
```

Comment:
It looks like this fragment is unnecessary.

Reply:
We removed the fragment.